web/themes/drupalnyc/templates/layout/html.html.twig

```diff
@@ -40,6 +40,7 @@
   <title>{{ head_title|safe_join(' | ') }}</title>
   <css-placeholder token="{{ placeholder_token }}">
   <js-placeholder token="{{ placeholder_token }}">
+  <script id="mcjs">!function(c,h,i,m,p){m=c.createElement(h),p=c.getElementsByTagName(h)[0],m.async=1,m.src=i,p.parentNode.insertBefore(m,p)}(document,"script","https://chimpstatic.com/mcjs-connected/js/users/8e760065071213ae345293359/1b686b1979e10a77ffb716515.js");</script>
```

Comment:
Prefer the Drupal way if possible: https://www.drupal.org/docs/8/api/javascript-api/add-javascript-to-your-theme-or-module